hash/vsmile_cart.xml

```diff
@@ -737,9 +737,9 @@ V.Smile Smartbook Smartidges (need a Smartbook touch tablet connected to a regul
 			<feature name="cart_type" value="lilac" />
 			<feature name="u1" value="" /> <!-- EPOXY BLOB ROM --> <!-- 2 globs, uses /CS2 for 2nd glob -->
 			<feature name="u2" value="" /> <!-- EPOXY BLOB ROM -->
-			<dataarea name="rom" size="0x0800000">
-				<rom name="alphaprkr2.u1" size="0x0400000" crc="4e635c94" sha1="96792d607a94f8fb36dbfff8c68f720c68c742a8" offset="0x0000000" />
-				<rom name="alphaprkr2.u2" size="0x0200000" crc="2feee80a" sha1="d1ca700602c3b2bce2a8920f017370cb0ef322de" offset="0x0400000" />
+			<dataarea name="rom" size="0x600000">
+				<rom name="alphaprkr2.u1" size="0x400000" crc="4e635c94" sha1="96792d607a94f8fb36dbfff8c68f720c68c742a8" offset="0x000000" />
+				<rom name="alphaprkr2.u2" size="0x200000" crc="2feee80a" sha1="d1ca700602c3b2bce2a8920f017370cb0ef322de" offset="0x400000" />
```

Comment:
Are you sure this won’t cause a problem?  The slot code may assume a minimum size for the ROM region.  Also, if you’re going to mess with this stuff, please put the offsets next to the sizes so they’re easier to see.